### PR TITLE
VBM-241: "make a post" fields naming changed

### PR DESCRIPTION
**Pull request by VladimirBludov**
1. naming fields changed
2. required images added
3. date picker changed
4. unnecessary fields in the information about the auction are removed

### CODE REVIEW

File: packages/mobile/package-lock.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
         "react-native-image-picker": "4.10.0",
         "react-native-linear-gradient": "2.6.2",
         "react-native-mmkv": "2.4.3",
+        "react-native-modal-datetime-picker": "^14.0.0",
```
Comment (AleksandrTermenzhy) [inline packages/mobile/package-lock.json:38]: --save-exact

File: packages/mobile/src/common/enums/ui/date-time-picker-type.enum.ts [hunk 2]
```diff
@@ -1,11 +1,13 @@
 enum DateTimeType {
   DATE = 'date',
   TIME = 'time',
+  DATE_TIME = 'datetime',
 }
 
 enum DateTimeFormat {
   DATE_ONLY = 'DD/MM/YYYY',
   TIME_ONLY = 'HH:mm',
+  DATE_TIME = 'DD/MM/YYYY HH:mm',
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/common/enums/ui/date-time-picker-type.enum.ts:10]: Pls combine it under one enum with DateFormat

File: packages/mobile/src/helpers/date/get-text-value-date/get-text-value-date.helper.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import dayjs from 'dayjs';
+import { DateTimeFormat, DateTimeType } from '~/common/enums/ui/ui';
+
+const getTextValueDate = (value: string, mode: DateTimeType): string | null => {
+  const date = value ? dayjs(value).format(DateTimeFormat.DATE_ONLY) : null;
+  const time = value ? dayjs(value).format(DateTimeFormat.TIME_ONLY) : null;
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/helpers/date/get-text-value-date/get-text-value-date.helper.ts:6]: What's the purpose of this helper?

File: packages/mobile/src/components/components.ts [hunk 4]
```diff
@@ -12,6 +12,7 @@ export {
 export { SafeAreaView } from 'react-native-safe-area-context';
 export { default as Popover } from 'react-native-popover-view';
 export { default as DateTimePicker } from '@react-native-community/datetimepicker';
+export { default as DateTimePickerModal } from 'react-native-modal-datetime-picker';
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/components/components.ts:15]: Why do we need both?

**Review by AleksandrTermenzhy — APPROVED** [review]

File: packages/mobile/package-lock.json (hunk 1 above)
Comment (VladimirBludov) [reply packages/mobile/package-lock.json:38]: corrected

File: packages/mobile/src/common/enums/ui/date-time-picker-type.enum.ts (hunk 2 above)
Comment (VladimirBludov) [reply packages/mobile/src/common/enums/ui/date-time-picker-type.enum.ts:10]: corrected

File: packages/mobile/src/components/components.ts (hunk 4 above)
Comment (VladimirBludov) [reply packages/mobile/src/components/components.ts:15]: No, corrected

File: packages/mobile/src/helpers/date/get-text-value-date/get-text-value-date.helper.ts (hunk 3 above)
Comment (VladimirBludov) [reply packages/mobile/src/helpers/date/get-text-value-date/get-text-value-date.helper.ts:6]: corrected

**Review by mazurGit — APPROVED** [review]

**Pull request merged**